scripts/initialize_ebpf_repo.ps1

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation
```

Comment:
can you also add a build.ps1 here that does the same thing as cicd `msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} /p:Platform=${{env.BUILD_PLATFORM}} ${{env.BUILD_OPTIONS}} ${{env.SOLUTION_FILE_PATH}}` -- maybe hard code the SOLITION_FILE_PATH.

Reply:
I did think about that, but then I did not go for it since it would take time away to someone just willing to reset and work directly on VS...